src/assets/wise5/components/table/tabulator-table/tabulator-table.component.ts

```diff
@@ -109,12 +122,36 @@ export class TabulatorTableComponent implements OnChanges, AfterViewInit {
     return cell.getValue();
   }
 
+  private initializeRowSelection(): void {
+    if (this.enableRowSelection && !this.disabled) {
+      this.tabOptions.columns.unshift({
+        formatter: 'rowSelection',
+        titleFormatter: 'rowSelection',
+        hozAlign: 'center',
+        headerSort: false,
+        frozen: true,
+        cellClick: (e, cell) => {
+          cell.getRow().toggleSelect();
+        }
+      });
+    }
+  }
+
   private processChanges(changes: SimpleChanges): void {
-    if (changes['tabColumns']) {
+    if (changes['tabColumns'] && !changes['tabColumns'].isFirstChange()) {
       this.table.setColumns(this.setupColumns(this.tabColumns));
     }
-    if (changes['tabData']) {
+    if (changes['tabData'] && !changes['tabData'].isFirstChange()) {
       this.table.setData(this.tabData);
     }
   }
+
+  private setupRowSelection(): void {
+    if (this.selectedRowIndices != null && this.selectedRowIndices.length > 0) {
```

Comment:
Since we initialize ```selectedRowIndices``` to empty array at the top of the class, the null check is unnecessary?
```suggestion
    if (this.selectedRowIndices.length > 0) {
```